src/codons.py

```diff
@@ -65,6 +77,10 @@ def translate_codons(codons: list[str]) -> list[str]:
 
 
 def translate_dna(dna: str) -> str:
+    if translate_codons(split_codons(dna)) != None:
+        amino = "".join(translate_codons(codons))
+    else amino = None
```

Comment:
This is a syntax error, if I am not mistaken?

Reply:
yeah, it says that "codons" is not defined. But isn't it defined in line 47?

Reply:
Yes, but that is in a different function, so you don't have it here. You need to get it from `translate_codons()`.

The syntax error I expected is because Python doesn't allow you to write `else <statement>`. You need a colon and then a statement following it.